plugins/fs/src/scope.rs

```diff
@@ -39,10 +25,8 @@ pub enum Event {
 
 #[derive(Default)]
 pub struct Scope {
-    pub(crate) allowed: Mutex<Vec<PathBuf>>,
-    pub(crate) denied: Mutex<Vec<PathBuf>>,
-    event_listeners: Mutex<HashMap<EventId, EventListener>>,
-    next_event_id: AtomicU32,
+    // TODO: Remove Option in v2, just used to keep Default
```

Comment:
let's just remove this whole scope implementation and expose tauri's internal scope and break the APIs of this plugin in a minor version.

Reply:
that's what i wanted to hear

Reply:
or wait, what do you mean with the "expose tauri's internal scope" part?

Reply:
nvm I thought we have something similar to `Manager::asset_protocol_scope` but for general fs operations.